src/test/java/org/springframework/data/couchbase/repository/N1qlCrudRepositoryIntegrationTests.java

```diff
@@ -152,7 +153,12 @@ public void shouldFailConversionWithStringReturnType() {
   public void shouldDoNumericProjectionWithStringBasedQuery() {
     Party partyHasKeyword = new Party(KEY_PARTY_KEYWORD, "party", "desc is a N1QL keyword", new Date(), 4000000, new Point(500, 500));
     partyRepository.save(partyHasKeyword);
-
+    Optional<Party> p = null;
```

Comment:
Formatting looks a bit odd. It would make sense to reformat the codebase using Spring Data's formatting settings for a consistent code format (see https://github.com/spring-projects/spring-data-build/tree/master/etc/ide).